cranelift/reader/src/parser.rs

```diff
@@ -3678,6 +3678,7 @@ mod tests {
     }
 
     #[test]
+    #[cfg(feature = "riscv")]
```

Comment:
This feature doesn't exist.

Reply:
Hmm, the changes in `parser.rs` were unintended, sorry about that -- part of my experiments in removing the old backend altogether. (I'll put together a draft PR for that soon-ish.) Reverted this file.